portal/src/graphql/adminapi/mutations/updateIdentityMutation.generated.ts

```diff
@@ -24,13 +24,23 @@ export const UpdateIdentityMutationDocument = gql`
         edges {
           node {
             id
+            type
```

Comment:
The standard attributes are still stale. Please also fetch the standard attributes in this mutation result.

Reply:
Let me fix this.